docs/source/repository_structure.mdx

```diff
@@ -41,9 +80,29 @@ my_dataset_repository/
     └── validation.csv
 ```
 
-Note that if a file contains *test* but is embedded in another word (e.g. `testfile.csv`), it's not counted as a test file.
-It must be delimited by non-word characters, e.g. `test_file.csv`.
-Supported delimiters are underscores, dashes, spaces, dots and numbers.
+### Single split
+
+When 🤗 Datasets can't find any of the above patterns, then it'll treat all the files as a single train split. If your dataset splits aren't loading as expected, this may be why!
```

Comment:
```suggestion
When 🤗 Datasets can't find any of the above patterns, then it'll treat all the files as a single train split. If your dataset splits aren't loading as expected, it may be due to incorrect pattern.
```